src/models/filter.rs

```diff
@@ -166,6 +208,16 @@ impl SourceCodeUnit {
             return false;
           }
         }
+        for query_with_holes in filter.contains() {
```

Comment:
I am unable to comment a the appropriate location (because of limitation of Github UI I guess). ANyhoo, the documentation for the enclosing fn `_check` should be updated.